controllers/rcj_soccer_referee_supervisor/rcj_soccer_referee_supervisor.py

```diff
@@ -1,7 +1,14 @@
+from math import ceil
 from referee.consts import MATCH_TIME, TIME_STEP
 from referee.referee import RCJSoccerReferee
 
-referee = RCJSoccerReferee(match_time=MATCH_TIME)
+referee = RCJSoccerReferee(
+    match_time=MATCH_TIME,
+    progress_check_steps=ceil(15/(TIME_STEP/1000.0)),
```

Comment:
What about putting these `ceil(...)` values to consts.py?

Reply:
I must say I don't really like this `consts.py` approach that much. In my view, the functionality of the supervisor ought to be configurable from the `rcj_soccer_referee_supervisor.py` file -- you should not need to update anything else, much less another specific file which is deeper in another repository.

That said, this is certainly a temporary solution as I completely agree that it belongs to a constant, whether in `consts.py` or this particular file.

Reply:
Ok, no problem